stage_test_setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+kubectl apply -f config/samples/metrics_v1alpha1_floorplan.yaml
```

Comment:
Let's assume that there are some cronjobs already. Then we won't need to set this one up.